how_to_build.md

```diff
@@ -33,7 +33,7 @@ At the time of writing the build is only confirmed to work on Windows using the
 
     `gem install bundler`
 
-1. Install gems:
+1. Install gems  (ensuring you're in the project root):

```

Comment:
@medmondson to keep the language consistent with other parts of the document, could we change this to

> Install gems (after navigating to your clone root folder):

If you're happy to make that change, can you please amend the current commit rather than adding another one, and force push to your branch.